lib/kitchen/directions/bake_annotation_classes/v1.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Kitchen::Directions::BakeAnnotationClasses
+  class V1
+    def bake(chapter:)
+      chapter.search('p.annotation').each do |annotation|
+        annotation.wrap_children('span', class: 'os-text')
+        annotation.prepend(child:
+          <<~HTML
+            <div class="os-icons"></div>
+          HTML
+        )
+      end
+      annotation_icon_classes = %w[linguistic-icon
+                                   culture-icon
+                                   dreaming-icon
+                                   visual-icon
+                                   speech-icon
+                                   auditory-icon kinesthetic-icon]
+      annotation_icon_classes.each do |annotation_icon_class|
+        chapter.search("p.#{annotation_icon_class}").each do |annotation_with_icon_class|
```

Comment:
try this new feature! 

`chapter.search(annotation_icon_class.prefix('p.').join(', ')).each do |annotation_with_icon_class|`

Reply:
@KarinaMendez2 I tried to refactor the lines into one loop using the `prefix` feature, but got stuck when there are one that one icon classes in a paragraph which need to be places in separate `spans`: 
![image](https://user-images.githubusercontent.com/55203598/129052209-4ab68c79-38f6-45ba-a50d-d97f6f76a5a3.png)
![image](https://user-images.githubusercontent.com/55203598/129052102-4b26ce7a-ba4f-4e62-a3ce-44d9e09773c0.png)
I guess in this case we must keep the outer loop. If you will have some other idea please let me know :slightly_smiling_face: